Block/FormatterBlockService.php

```diff
@@ -23,7 +23,7 @@
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class FormatterBlockService extends BaseBlockService
+class FormatterBlockService extends AbstractBlockService
```

Comment:
Ok, this change (inheritance) broke usage of FormatterBlockService in Sonata sandbox when composing the page. 

It can not be seen in demo, since last commit is (as I can see) from December 8th 2016. https://github.com/sonata-project/sandbox/commits/2.4-develop 

However, BaseBlockService does not implements use Sonata\BlockBundle\Block\Service\AdminBlockServiceInterface and therefore it can not be used in Sonata page bundle in admin as drag&drop control.

Is this by design? What is alternative? Hope you didn't intended to remove this feature...